config.go

```diff
@@ -105,6 +105,10 @@ func newConfig(root string) (c config, err error) {
 		}
 
 		errMsg = errMsg + strings.Join(volumesErrors, "\n")
+		// we have errors from both volumes and envs sections -> let's make sure they are both indented
```

Comment:
There is a array of errors from `volumes` validation and from `envs` validation. The errors are not ended by CR, so in case we have problems (and thus error messages) in both sections, the two error strings will be directly concatenated. Therefore we need to put extra CR between.